environment/environment_templates.go

```diff
@@ -173,6 +204,60 @@ func NewChainlinkCluster(nodeCount int) K8sEnvSpecInit {
 		manifests: manifests,
 	}
 
+	return newChainlinkManifest(chainlinkCluster, "basic-chainlink")
+}
+
+// NewMixedVersionChainlinkCluster mixes the currently latest chainlink version (as defined by the config file) with
+// the past two stable versions, ensuring that at least one of each is deployed
+func NewMixedVersionChainlinkCluster(nodeCount int) K8sEnvSpecInit {
```

Comment:
We should add unit tests for this function. It may also make sense to break the github client part out into its own block to make testing easier (mocking a function instead of mocking a client)